src/entities/createPool/createPoolV3/gyroECLP/createPoolGyroECLP.ts

```diff
@@ -79,6 +80,31 @@ export class CreatePoolGyroECLP implements CreatePoolBase {
     }
 }
 
+// We cannot just sort the tokens, but we have to update the ECLP params, too, to preserve their meaning!
+export function sortECLPInputByTokenAddress(
+    input: CreatePoolGyroECLPInput,
+): CreatePoolGyroECLPInput {
+    if (
+        input.tokens[0].address.toLowerCase() <
+        input.tokens[1].address.toLowerCase()
+    ) {
+        return input;
+    } else {
+        const D18 = 10n ** 18n; // 18 decimal precision is how params are stored
+        return {
+            ...input,
+            tokens: [input.tokens[1], input.tokens[0]],
+            eclpParams: {
+                alpha: (D18 * D18) / input.eclpParams.beta,
+                beta: (D18 * D18) / input.eclpParams.alpha,
+                c: input.eclpParams.s,
+                s: input.eclpParams.c,
+                lambda: input.eclpParams.lambda,
+            },
+        };
+    }
+}
```

Comment:
currently, the pool creation UI enforces the token order to be lexicographical and then allows user to "invert" chart using similar logic [here](https://github.com/balancer/pool-creator/blob/66febd8327a21826f06c2f2c433d97c23b982218/packages/nextjs/utils/gryo/helpers.ts#L38-L59) 

i do like the idea of handling within the SDK instead since it could help simplify the creation UI :thinking: